util/src/test-support/java/tech/pegasys/artemis/util/Waiter.java

```diff
@@ -32,14 +32,18 @@
   private static final Duration INITIAL_POLL_INTERVAL = Duration.ofMillis(200);
   private static final Duration MAX_POLL_INTERVAL = Duration.ofSeconds(5);
 
-  public static void waitFor(final Condition assertion) {
-    Awaitility.waitAtMost(DEFAULT_TIMEOUT_SECONDS, TimeUnit.SECONDS)
+  public static void waitFor(final Condition assertion, int timeoutSeconds) {
```

Comment:
I'd pass the `TimeUnit` in as a parameter as well.  While we'll almost always use seconds, it makes the calling code a lot clearer.

Reply:
done.